### PR TITLE
FunctionOperator fixes

### PR DESCRIPTION
**Pull request by vpuri3**
fixed update_coefficients-bang, added getops, isconstant, for functionop

pls bump version after this. thanks

### CODE REVIEW

File: src/interface.jl [hunk 1]
```diff
@@ -208,6 +208,8 @@ expmv!(v,L::AbstractSciMLOperator,u,p,t) = mul!(v,exp(L,t),u)
 # fallback implementations
 ###
 
+#Base.size(L::AbstractSciMLOperator) = size(convert(AbstractMatrix, L))
```
Comment (ChrisRackauckas) [inline src/interface.jl:211]: clean please

Comment (vpuri3) [reply src/interface.jl]: done

File: src/func.jl [hunk 2]
```diff
@@ -171,7 +171,13 @@ function FunctionOperator(op,
                     )
 end
 
+# TODO - update_coefficients(L, u, p, t)
```
Comment (ChrisRackauckas) [inline src/func.jl:174]: Why todo? This is here?

Comment (vpuri3) [reply src/func.jl:174]: I implemented `update_coefficients<BANG>`. the todo is for `update_coefficients`.

Comment (vpuri3) [reply src/func.jl:174]: hold on lemme stop being lazy and implement `update_coefficients` as well

Comment (vpuri3) [reply src/func.jl]: done

**Comment by vpuri3** [comment]
@ChrisRackauckas tests passed. This is good to go as well

**Comment by ChrisRackauckas** [comment]
Can you follow this up with the correctness fix?

**Comment by vpuri3** [comment]
> Can you follow this up with the correctness fix?

@ChrisRackauckas what are you referring to?

**Comment by ChrisRackauckas** [comment]
Removing the caches in https://github.com/SciML/SciMLOperators.jl/issues/145

**Pull request merged**